packages/round-manager/src/features/round/__tests__/ApplicationsApproved.test.tsx

```diff
@@ -115,31 +107,30 @@ describe("<ApplicationsApproved />", () => {
     })
   })
 
-  describe("when bulkSelect is true", () => {
-    it("renders reject buttons on each project card", () => {
-      renderWrapped(<ApplicationsApproved bulkSelect={true} />);
+  describe("when bulk select is active", () => {
+    it("renders reject option on each project card", () => {
+      setupInBulkSelectionMode();
       expect(screen.queryAllByTestId("reject-button"))
         .toHaveLength(grantApplications.length);
     });
 
-    it("does not display approve buttons in approved applications tab", () => {
-      renderWrapped(<ApplicationsApproved bulkSelect={true} />)
-
+    it("does not display approve options in approved applications tab", () => {
+      renderWrapped(<ApplicationsApproved />)
```

Comment:
should do `setupInBulkSelectionMode();` here too, since it's inside the "when bulk select is active" describe block